### DISCUSSION TITLE
Change the default build number, shown by github

### DISCUSSION THREAD

**Discussion by conajma** [community/community#48020] [Actions]
### Select Topic Area

Product Feedback

### Body

![Reference Image](https://user-images.githubusercontent.com/100670044/220053636-4839f0fd-419a-4a8f-a6e1-2cce55345944.png)
Hi all,
Can we change this build number here, which is #25 as of now, Can I make it customized? Like I want to append 2.2.0 for example.
The build number should be 2.2.0.25 at the end?

Is this possible here?

**Comment by trwstin** [comment]
I don't think that's the build number... more like the n-th operation in the workflow?

However, you can do something similar to having build numbers by creating a Release (right side of the Repo) and editing the version number to your liking:
![image](https://user-images.githubusercontent.com/68576751/220081590-2ac4e204-576c-4fe0-8171-f288dc881731.png)

**Reply by conajma** [reply]
Thats the github runner number accessible by ${{ github.run_number }}
